docs/testingprocedures.md

```diff
@@ -0,0 +1,52 @@
+# Testing Procedures:
+
+## Before leaving
+
+### Packing list 
+* Power Strip
+* Table
+* Extension Cord
+* Battery Box
+* Battery charger
+* Tether
+* Network Box
+* Ethernet Cables (2 + #of devs)
+* Laptops
+* 7/16 wrench
+* 5/32 Allen Key
+* Allen Key for paintball tank
+* Pliers
+* Flat-head screwdriver
+* Kill wand
+* Goggles / Snorkeling Tube
+* Pool noddles
+* Dive fins
+* O'ring grease
+* Zip ties
+* clippers
+* Towels
+
+
+### Software Checklist
+* Update and build code
+* Verify all sensors output data
+* Verify thrusters spin given command
+* Verify kill (soft an hard)
+
+
+### At testing site
+
+* Get wall power to powerstrip
+* Setup and connect to Network Box
+* Roll tether and connect it ot network box **(DO NOT USE POE)**
+* Connect Sub to to tether **(NOT POE)**
+* Connect battery alarm, power on sub
+* SSH into sub
+* Start tmux, write code
+* Should add Grease all Electrical connections with appropriate grease
```

Comment:
should be
` Grease all electrical connections`
And under
`Before leaving`

Reply:
Fixed it